### PR TITLE
AEM 6.1 - Update Grabbit 4.x build so that OOTB dependencies are properly excluded

### PR DESCRIPTION
**Pull request by sagarsane**
Fix for #79 

@jbornemann @jdigger @jazeren1 please take a look. To test, install / uninstall this Grabbit build a bunch of times and each time, it should be MUCH smoother at doing so.

Also, since you will likely have another version of Grabbit already installed, please : 

1. Delete `/apps/grabbit/install` & `/etc/packages/twc/Grabbit.zip` from CRXDE
2.  Wait for AEM to stabilize (look in the logs and /system/console/bundles should also show all active)
3. Restart AEM
4. Then install this build
5. Try install and uninstall of this build a bunch of times

### CODE REVIEW

**Comment by sagarsane** [comment]
The list of jars in Grabbit build is much smaller now : 

```
/apps/grabbit/install/grabbit-4.0.5.jar  
/apps/grabbit/install/commons-io-1.4.jar  
/apps/grabbit/install/com.springsource.org.aopalliance-1.0.0.jar  
/apps/grabbit/install/org.apache.servicemix.bundles.protobuf-java-2.4.1_1.jar  
/apps/grabbit/install/org.apache.servicemix.bundles.spring-aop-3.2.9.RELEASE_1.jar  
/apps/grabbit/install/org.apache.servicemix.bundles.spring-beans-3.2.9.RELEASE_1.jar  
/apps/grabbit/install/org.apache.servicemix.bundles.spring-context-3.2.9.RELEASE_1.jar  
/apps/grabbit/install/org.apache.servicemix.bundles.spring-core-3.2.9.RELEASE_1.jar  
/apps/grabbit/install/org.apache.servicemix.bundles.spring-expression-3.2.9.RELEASE_1.jar  
/apps/grabbit/install/org.apache.servicemix.bundles.spring-tx-3.2.9.RELEASE_1.jar  
/apps/grabbit/install/groovy-all-2.3.6.jar  
/apps/grabbit/install/jettison-1.1.jar  
/apps/grabbit/install/javax.jms-3.1.1.jar  
/apps/grabbit/install/spring-batch-core-2.2.7.RELEASE.jar  
/apps/grabbit/install/spring-batch-infrastructure-2.2.7.RELEASE.jar  
/apps/grabbit/install/spring-osgi-core-2.0.0.M1.jar  
/apps/grabbit/install/spring-osgi-extender-2.0.0.M1.jar  
/apps/grabbit/install/spring-osgi-io-2.0.0.M1.jar  
/apps/grabbit/install/spring-retry-1.0.2.RELEASE.jar  
```

File: gradle.properties [hunk 1]
```diff
@@ -5,7 +5,7 @@ org.gradle.jvmargs=-XX:MaxPermSize=512m -XX:+CMSClassUnloadingEnabled -XX:+CMSPe
 bundleInstallRoot = /apps/grabbit/install
 
 group = com.twcable.grabbit
-version = 4.0.4
+version = 4.0.5
```
Comment (jdigger) [inline gradle.properties]: Version changes should happen as their own commit per [RELEASING.adoc](../../blob/master/docs/RELEASING.adoc)

Comment (sagarsane) [inline gradle.properties]: okay. yeah.

**Comment by jdigger** [comment]
+1 CR and testing

Once the bogus `version` change is removed, ready to be brought into master

**Comment by jazeren1** [comment]
+1 Testing.  

Tested:
- Installation successful
- Successful runs of grabbit
- Multiple installs/uninstalls/grabbit runs

**Pull request merged**